contracts/token/LinearVestingWithPenalty.sol

```diff
@@ -27,13 +27,12 @@ contract LinearVestingWithPenalty is Initializable, ERC20Upgradeable, AccessCont
     bytes32 public constant SCHEDULE_MANAGER_ROLE = keccak256("SCHEDULE_MANAGER_ROLE");
     bytes32 public constant UPGRADER_ROLE = keccak256("UPGRADER_ROLE");
 
-    IERC20 public sophtoken;        // The underlying token being vested
-    address public penaltyRecipient;           // Address receiving penalties
-    uint256 public vestingStartDate;           // Global vesting start date
-    uint256 public penaltyPercentage;          // Penalty percentage for early withdrawals (e.g., 50 for 50%)
+    IERC20 public sophtoken;                       // The underlying token being vested
```

Comment:
writing comment here because it's not part of the diff, just a nit pick and up to you but
`getVestingSchedules(address beneficiary) external view returns (VestingSchedule[] memory)` wouldn't be needed since `vestingSchedules` is already public

Reply:
fixed

Reply:
fixed